util.lua

```diff
@@ -209,6 +209,28 @@ local function findCityByEntityUnitNumber(unitNumber)
     return cityName or "Unknown"
 end
 
+local function list_special_city_buildings(city, name)
+    local entities = {}
+    if city.special_buildings.other[name] ~= nil and #city.special_buildings.other[name] > 0 then
+        entities = city.special_buildings.other[name]
+    else
+        entities = game.surfaces[city.surface_index].find_entities_filtered{
+            name=name,
+            position=city.special_buildings.town_hall.position,
```

Comment:
oh, how this `LuaEntity` access got into here, again?! this:
```diff
-position=city.special_buildings.town_hall.position,
+position=city.center,
```

Reply:
That branch is a couple weeks old, so there may be some old stuff.